### DISCUSSION TITLE
Sort topic comments in addition to topics

### DISCUSSION THREAD

**Discussion by darsnack** [community/community#3376] [Discussions] [answered]
It would be nice to sort topic comments by metrics like upvotes as well. For example, if there is a long standing design discussion, it would be nice to use the upvote mechanism to see the more pertinent sub-discussions towards the top.

**Comment by PranamBhat** [comment, marked as answer]
Hi,

Choosing "Past week", "Past month", "Past year" and "All" will gives you a list of discussions based on the "Vote Counts".

[https://user-images.githubusercontent.com/32120033/108022423-8701dd00-7046-11eb-9a4b-649d0cc998bb.JPG](https://user-images.githubusercontent.com/32120033/108022423-8701dd00-7046-11eb-9a4b-649d0cc998bb.JPG)

**Reply by darsnack** [reply]
I was referring to the comments within a top level discussion. Appears that "Newest", "Oldest", and "Top" exist though, so that solves my issue. Thanks!
